js/main.js

```diff
@@ -0,0 +1,34 @@
+/* global data */
+const $input = document.querySelector('.photo-URL');
+const $image = document.querySelector('img');
+
+console.log($input);
+console.log($image);
+
+$input.addEventListener('input', function (event) {
+  $image.src = $input.value;
+});
+
+const $submit = document.querySelector('form');
+console.log($submit);
+const $title = document.querySelector('#title-box');
+console.log($title);
+
+$submit.addEventListener('submit', function (event) {
+  event.preventDefault();
+  const newData = {
+    title: $title.value,
+    notes: document.getElementById('notes').value,
+    imageUrl: $image.src,
+    entryId: data.nextEntryId,
+  };
+  console.log(newData);
+  data.nextEntryId++;
+  data.entries.push(newData);
```

Comment:
The unshift method should be used to push the new object to the front of the array per the directions:
**adds the new object to the beginning of the data model's array of entries.**

`data.entries.unshift(newData)`